torch/csrc/api/include/torch/nn/modules/sequential.h

```diff
@@ -298,6 +308,22 @@ class SequentialImpl : public Cloneable<SequentialImpl> {
   }
 
  private:
+  /// Matches `Sequential(std::string("m1"), Module(1), ...)` case
+  template <typename Module, typename... Rest>
+  void push_back(std::string name, Module&& module, Rest&&... rest) {
+    auto index = add_to_modules(module);
+    register_module(name, modules_[index].ptr());
+    push_back(std::forward<Rest>(rest)...);
+  }
+
+  /// Matches `Sequential("m1", Module(1), ...)` case
+  template <typename Module, typename... Rest>
+  void push_back(const char* name, Module&& module, Rest&&... rest) {
```

Comment:
You can remove this overload and the whole `const char*` business, it's not worth it